install.sh

```diff
@@ -301,8 +331,8 @@ docker-compose down
 
 echo "Building frontend."
 # build frontend with updated env file
-cp square-frontend/.env.production square-frontend/.env.production-backup
-sed -e "s/%%SQUARE_URL%%/https:\/\/$SQUARE_URL/g" square-frontend/.env.template > square-frontend/.env.production
+cp frontend/.env.production frontend/.env.production-backup
+sed -e "s/%%SQUARE_URL%%/https:\/\/$SQUARE_URL/g" frontend/.env.template > frontend/.env.local
```

Comment:
why is this `.local`? The `.production` actually has a meaning, because when the frontend is built it will use that file (this is something vue specific)

Reply:
I am not sure why but this line here: https://github.com/UKP-SQuARE/square-core/blob/2dd9b0846b3f5426006e0a58f962631d94acf573/frontend/Dockerfile#L12-L13 

which builds the frontend app fails when .env.production is used and works when .env.local is used. I guess which .env file to use depends on the `mode` argument (which in local deployment is `local`).